pkg/agreementreceiver/server.go

```diff
@@ -51,16 +85,17 @@ func (s *Server) BandwidthAgreements(stream pb.Bandwidth_BandwidthAgreementsServ
 		case err := <-errch:
 			return err
 		case <-ctx.Done():
+			s.logger.Debug("ctx<-Done()")
 			return nil
 		case agreement := <-ch:
 			go func() {
-				fmt.Println(agreement)
-				//TODO: write to DB
-				//err = s.DB.WriteAgreement(agreement)
-				// if err != nil {
-				// 	return err
-				// }
+				s.logger.Debug("about to create a postgres entry")
```

Comment:
Needs a better log message to indicate what the entry contains

Reply:
removed the logger info for now as it serves no useful purpose.It is there for any clean up action if needed, then it has to be done here like handling Ctrl+c kind of things